sssd_test_framework/hosts/keycloak.py

```diff
@@ -0,0 +1,62 @@
+"""Keycloak multihost host."""
+
+from __future__ import annotations
+
+from .base import BaseDomainHost
+
+__all__ = [
+    "KeycloakHost",
+]
+
+
+class KeycloakHost(BaseDomainHost):
+    """
+    Keycloak host object.
+
+    Provides features specific for Keycloak server.
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.adminpw = self.config.get('adminpw', 'Secret123')
+
+        # Backup of original data
+        self.__backup: str = None
+
+    def kclogin(self) -> None:
+        """
+        Obtain ``admin`` user credentials for Keycloak
+        """
+        #self.ssh.exec(['kinit', 'admin'], input=self.adminpw)
+        self.ssh.exec(['/opt/keycloak/bin/kcadm.sh', 'config', 'credentials',
+                       '--server', f'https://{self.hostname}:8443/auth/',
+                       '--realm', 'master', '--user', 'admin',
+                       '--password', self.adminpw])
+        return
```

Comment:
I think you can remove this, not returning anything.

Reply:
removing